static/src/js/form.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
Can we change this path into something else to be more recognizable for what it is? Example `src/apps/accounts/registration.jsx` ?